packages/assets/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "target": "es5",
+    "declaration": true,
+    "outDir": "./lib",
+    "sourceMap": true,
```

Comment:
Lurer på om source maps ikke er nødvendig i dette tilfellet. Kanskje ikke de gjør så mye skade heller da egentlig. 🤔

Reply:
Etter å ha funnet ut hva det er for noe, ser det ikke ut som det er nødvendig nei. Men hadde nok heller ikke gjort noe skade nei